lib/minio_runner/config.rb

```diff
@@ -2,14 +2,15 @@
 
 module MinioRunner
   class Config
-    attr_accessor :install_dir, :cache_time, :buckets, :public_buckets, :log_level
-    attr_accessor :minio_data_directory,
-                  :minio_root_user,
+    attr_accessor :cache_time, :buckets, :public_buckets, :log_level
+    attr_accessor :minio_root_user,
                   :minio_root_password,
                   :minio_domain,
                   :minio_port,
                   :minio_console_port
 
+    attr_reader :install_dir, :minio_data_directory
```

Comment:
`install_dir=` and `minio_data_directory=` methods are defined in this class and it triggered a rubocop rule about duplicated method definition.